package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.0.1",
   "description": "Simorgh",
   "scripts": {
-    "amp:validate": "wait-on http://localhost:7080/news/articles/c9rpqy7pmypo.amp && (amphtml-validator http://localhost:7080/news/articles/c9rpqy7pmypo.amp || true)",
+    "amp:validate": "wait-on http://localhost:7080/news/articles/c9rpqy7pmypo.amp && amphtml-validator http://localhost:7080/news/articles/c9rpqy7pmypo.amp",
```

Comment:
For the record, just because I had to Google this - the [`|| true` is to prevent this from being a failing step](https://unix.stackexchange.com/questions/325705/why-is-pattern-command-true-useful). We now want it to be a failing step, so have removed it.